gitlab/v4/objects.py

```diff
@@ -2069,27 +2069,24 @@ class ProjectForkManager(CreateMixin, ListMixin, RESTManager):
     )
     _create_attrs = (tuple(), ("namespace",))
 
-    def list(self, **kwargs):
-        """Retrieve a list of objects.
+    def create(self, data, **kwargs):
+        """Creates a new object.
 
         Args:
-            all (bool): If True, return all the items, without pagination
-            per_page (int): Number of items to retrieve per request
-            page (int): ID of the page to return (starts with page 1)
-            as_list (bool): If set to False and no pagination option is
-                defined, return a generator instead of a list
+            data (dict): Parameters to send to the server to create the
+                         resource
             **kwargs: Extra options to send to the server (e.g. sudo)
 
-        Returns:
-            list: The list of objects, or a generator if `as_list` is False
-
         Raises:
             GitlabAuthenticationError: If authentication is not correct
-            GitlabListError: If the server cannot perform the request
-        """
+            GitlabCreateError: If the server cannot perform the request
 
-        path = self._compute_path("/projects/%(project_id)s/forks")
-        return ListMixin.list(self, path=path, **kwargs)
+        Returns:
+            RESTObject: A new instance of the managed object class build with
+                the data sent by the server
+        """
+        path = self.path[:-1]  # drop the 's'
```

Comment:
Maybe we could use the real path instead to avoid confusion.

```
/projects/%(project_id)s/fork
```

Reply:
I am just trusting that the person who did the pipeline manager knew what he/she was doing. I suspect that if I replace the path as above, it won't substitute in for `%(project_id)s` which was the cause for the 404 in the earlier `ProjectForkManager.list` method implementation.

Reply:
ah you're right. Sorry.